exonum/src/runtime/mod.rs

```diff
@@ -229,6 +229,11 @@ pub trait Runtime: Send + Debug + 'static {
 
     /// Calls `before_commit` for all the services stored in the runtime.
     ///
+    /// # Notes for runtime developers.
+    ///
+    /// * The order of services during the invocation this method must be the same for each node;
+    /// in other words the order of runtime services must be deterministic.
```

Comment:
'the same for each node' sounds stronger than 'deterministic', therefore, the latter clause might be dropped.